forum/static/forum/jquery.js

```diff
@@ -0,0 +1,4 @@
+$(document).ready(function () {
+    alert('starting reddit clone');
+    // Add in more as required
+})
```

Comment:
You should probably remove this file or at least comment it out. Also, `jquery.js` is not a good name.

Reply:
will comment out for now